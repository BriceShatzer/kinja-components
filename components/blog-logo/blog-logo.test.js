import * as React from 'react';
import { render } from 'enzyme';

import BlogLogo from './blog-logo';

describe('<BlogLogo />', () => {
	it('should render avclub logo', () => {
		expect(render(<BlogLogo name="avclub" />)).toMatchSnapshot();
	});

	it('should render avclub logo with forced 2x scale', () => {
		expect(<BlogLogo name="avclub" scale="2" />).toMatchSnapshot();
	});
});
