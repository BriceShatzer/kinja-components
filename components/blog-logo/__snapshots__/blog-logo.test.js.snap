// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BlogLogo /> should render avclub logo 1`] = `
.c0 {
  --scale: 1;
}

.c1 {
  position: relative;
  height: calc(32px * var(--scale));
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-flex: 0;
  -webkit-flex-grow: 0;
  -ms-flex-positive: 0;
  flex-grow: 0;
  -webkit-flex-shrink: 0;
  -ms-flex-negative: 0;
  flex-shrink: 0;
}

.c1 svg {
  position: absolute;
  -webkit-transform-origin: top left;
  -ms-transform-origin: top left;
  transform-origin: top left;
  -webkit-transform: scale(var(--scale));
  -ms-transform: scale(var(--scale));
  transform: scale(var(--scale));
}

.c1.avclub {
  width: calc(150px * var(--scale));
}

.c1.avclub svg {
  top: -40%;
}

.c1.clickhole {
  width: calc(216px * var(--scale));
}

.c1.clickhole svg {
  top: -4%;
}

.c1.deadspin {
  width: calc(254px * var(--scale));
}

.c1.deadspin svg {
  top: 0%;
}

.c1.earther {
  width: calc(217px * var(--scale));
}

.c1.earther svg {
  top: -1%;
}

.c1.gizmodo {
  width: calc(225px * var(--scale));
}

.c1.gizmodo svg {
  top: 0%;
}

.c1.jalopnik {
  width: calc(228px * var(--scale));
}

.c1.jalopnik svg {
  top: 0%;
}

.c1.jezebel {
  width: calc(206px * var(--scale));
}

.c1.jezebel svg {
  top: 0%;
}

.c1.kotaku {
  width: calc(212px * var(--scale));
}

.c1.kotaku svg {
  top: -10%;
}

.c1.lifehacker {
  width: calc(180px * var(--scale));
}

.c1.lifehacker svg {
  top: -25%;
}

.c1.splinter {
  width: calc(261px * var(--scale));
}

.c1.splinter svg {
  top: -7%;
}

.c1.theinventory {
  width: calc(258px * var(--scale));
}

.c1.theinventory svg {
  top: -13%;
}

.c1.theroot {
  width: calc(205px * var(--scale));
}

.c1.theroot svg {
  top: -20%;
}

.c1.theonion {
  width: calc(262px * var(--scale));
}

.c1.theonion svg {
  top: -22%;
}

.c1.thetakeout {
  width: calc(140px * var(--scale));
}

.c1.thetakeout svg {
  top: -34%;
}

.c1.kinja {
  width: calc(152px * var(--scale));
}

.c1.kinja svg {
  top: -22%;
}

.c1.gomedia {
  width: calc(241px * var(--scale));
}

.c1.gomedia svg {
  top: -20%;
}

.c1.kinjadeals {
  width: calc(203px * var(--scale));
}

.c1.kinjadeals svg {
  top: -10%;
}

.c1.kinjavideo {
  width: calc(243px * var(--scale));
}

.c1.kinjavideo svg {
  top: -6%;
}

<div
  class="c0"
>
  <div
    class="avclub c1"
  >
    <svg
      aria-label="TheAVClub logo"
      height="59"
      viewBox="0 0 150 59"
      width="150"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M83.87 37.92a12.85 12.85 0 0 1-9.28 3.72c-3.62 0-6.7-1.24-9.27-3.72a12.04 12.04 0 0 1-3.84-8.98c0-3.5 1.28-6.5 3.84-8.98a12.86 12.86 0 0 1 9.27-3.72c3.63 0 6.72 1.24 9.28 3.72l-3.55 3.44a7.99 7.99 0 0 0-5.73-2.28c-2.22 0-4.13.76-5.7 2.3a7.42 7.42 0 0 0-2.38 5.52c0 2.16.8 4 2.38 5.53a7.92 7.92 0 0 0 5.71 2.3c2.23 0 4.14-.76 5.73-2.29l3.54 3.44zm20.27 2.66H87.07V16.24h5.07v19.53h12v4.81zm22.4-9.88c0 3.02-1.1 5.6-3.28 7.74a10.91 10.91 0 0 1-7.92 3.2c-3.1 0-5.74-1.07-7.92-3.2a10.42 10.42 0 0 1-3.28-7.74V16.24h5.06V30.7c0 1.65.6 3.06 1.8 4.23a5.97 5.97 0 0 0 4.34 1.76c1.69 0 3.13-.58 4.33-1.76a5.7 5.7 0 0 0 1.8-4.23V16.24h5.07V30.7zm15.19-14.46c2.02 0 3.75.72 5.18 2.15a7.02 7.02 0 0 1 2.15 5.16c0 1.65-.5 3.13-1.49 4.44a7.6 7.6 0 0 1 2.43 5.74c0 2.17-.78 4.03-2.32 5.58a7.54 7.54 0 0 1-5.55 2.33H130.8v-25.4h10.93zm.3 14.82h-5.9v5.29h5.9c.74 0 1.36-.25 1.87-.76a2.63 2.63 0 0 0-.01-3.72 2.5 2.5 0 0 0-1.86-.81zm.26-9.53h-6.16v4.24h6.16c.65 0 1.21-.21 1.68-.63a1.93 1.93 0 0 0 0-2.98 2.43 2.43 0 0 0-1.68-.63zM47.26 5.94a29.5 29.5 0 1 1-35.5 47.12 29.5 29.5 0 1 1 35.5-47.12zM31.93 16.86h-5.58l11.71 24.23 11.47-24.23h-5.35l-6.23 13.5-6.02-13.5zm-10.3 0L9.48 41.09h5.28l1.46-3.17h9.97l1.23 3.17h5.25L21.63 16.86zm.08 11.59l2.53 5.26h-5.27l2.74-5.26z"
        fill="#1C263C"
        fill-rule="evenodd"
      />
    </svg>
  </div>
</div>
`;

exports[`<BlogLogo /> should render avclub logo with forced 2x scale 1`] = `
<BlogLogo
  name="avclub"
  scale="2"
/>
`;
